apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -109,25 +108,19 @@ export const gradeFormatter = function (grade, {rowData}) {
 
 export const loginInfoFormatter = function (loginType, {rowData}) {
   let sectionCode = '';
-  if (rowData.providerManaged) {
-    sectionCode = (
-      <div data-tip={i18n.providerManagedSection({provider: rowData.loginType})}>
-        {i18n.none()}
-        &nbsp;
-        <i
-          className="fa fa-question-circle"
-          style={styles.sectionCodeNone}
-        />
-        <ReactTooltip
-          role="tooltip"
-          effect="solid"
-        />
-      </div>
-    );
+  let pegasusUrl = pegasus('/teacher-dashboard#/sections/' + rowData.id + '/print_signin_cards');
```

Comment:
nit: looks like this could be const instead of let

Also, this is somewhat personal preference, but in case you're not familiar with it, this could be a good opportunity to use string interpolation, i.e.
```js
const pegasusUrl = pegasus(`/teacher-dashboard#/sections/${rowData.id}/print_signin_cards`);
```